src/components/Toast/Toast.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Canvas, Story, Props } from "@storybook/addon-docs";
+import {Toast} from "./Toast";
+
+<Meta
+  title="Catalog/Toast"
+  component={Toast}
+  args={{ message: "Error", type: "error" }}
+/>
+
+export const Template = (args) => {
+  return <Toast  {...args} />;
+};
+
+# Toast
+
+#### Toast component for showing alerts.
+
+# Example
+
+<Canvas>
+  <Story name="Toast">{Template.bind({})}</Story>
```

Comment:
**suggestion/thought:** For the `mdx` stories, we're hoping to use these more as guides on how to use a component. This gets trickier when showing off a component that we'd expect to use via a hook. In that mindset, I'd expect that in the only time we display the "Show code", it'd include something like:
```
const { showToast } = useToast();
return <Button onClick={() => showToast(...args) } label="Trigger Toast" />
```